devicetypes/smartthings/springs-window-fashions-shade.src/springs-window-fashions-shade.groovy

```diff
@@ -72,7 +72,7 @@ metadata {
         }
 
         valueTile("battery", "device.battery", decoration: "flat", inactiveLabel: false, width: 2, height: 2) {
-            state "battery", label:'battery', unit:"",
+            state "battery", label:'batt.', unit:"",
```

Comment:
Is there also a ticket opened for the app cutting off the text? That probably affects a bunch of devices, right?

Reply:
ICP-3152

Reply:
That's just for this one. Don't know about a "general" one. I mentioned in the commit message that the ideal solution would just be a battery icon, but we actually don't have one.